src/components/Modal/VaultStrategyModal/lib/VaultStrategyModalConcentratorApr.tsx

```diff
@@ -12,31 +12,21 @@ export const VaultStrategyModalConcentratorApr: FC<VaultProps> = (props) => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const data = breakdownApr.data as any
 
-  if (data?.totalApr !== undefined)
-    items.push({
-      label: "Total APR",
-      value: data?.totalApr,
-      emphasis: true,
-      disabledTooltip: true,
-    })
-  if (data?.baseApr !== undefined)
-    items.push({ label: "Base APR", value: data?.baseApr })
-  if (data?.crvApr !== undefined)
-    items.push({ label: "CRV APR", value: data?.crvApr })
-  if (data?.cvxApr !== undefined)
-    items.push({ label: "CVX APR", value: data?.cvxApr })
-  if (data?.extraRewardsApr !== undefined)
-    items.push({ label: "Extra Rewards APR", value: data?.extraRewardsApr })
-  if (data?.BALApr !== undefined)
-    items.push({ label: "BAL APR", value: data?.BALApr })
-  if (data?.swapFeeApr !== undefined)
-    items.push({ label: "Swap Fee APR", value: data?.swapFeeApr })
-  if (data?.AuraApr !== undefined)
-    items.push({ label: "Aura APR", value: data?.AuraApr })
-  if (data?.GMXApr !== undefined)
-    items.push({ label: "GMX APR", value: data?.GMXApr })
-  if (data?.ETHApr !== undefined)
-    items.push({ label: "ETH APR", value: data?.ETHApr })
+  items.push({
```

Comment:
Now that its unconditional, create the array once with all of the data instead of pushing each item individually:

```ts
const items = [
  { label: "Total APR", ... },
  ...
]
```

Reply:
right